cnf-certification-test/lifecycle/scaling/scaling_helper.go

```diff
@@ -0,0 +1,36 @@
+package scaling
+
+import (
+	"github.com/test-network-function/cnf-certification-test/pkg/configuration"
+	apiv1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	scalingv1 "k8s.io/api/autoscaling/v1"
+)
+
+func GetResourceHPA(hpaList []*scalingv1.HorizontalPodAutoscaler, name, namespace, kind string) *scalingv1.HorizontalPodAutoscaler {
+	for _, hpa := range hpaList {
+		if hpa.Spec.ScaleTargetRef.Kind == kind && hpa.Spec.ScaleTargetRef.Name == name && hpa.Namespace == namespace {
+			return hpa
+		}
+	}
+	return nil
+}
+func IsManaged(depName string, managedDep []configuration.ManagedDeploymentsStatefulsets) bool {
+	for _, dep := range managedDep {
+		if dep.Name == depName {
```

Comment:
I would do a string contains here to give some flexibility : strings.contains(depName,dep.Name)

Reply:
mmm im not sure about that..
if we are searching the same statefulset/deployment so why not to check equal?